chartpress.yaml

```diff
@@ -9,14 +9,14 @@ charts:
       hub:
         valuesPath: hub.image
         buildArgs:
-          JUPYTERHUB_VERSION: 0.8.1
+          JUPYTERHUB_VERSION: 0.9.0b1
```

Comment:
Is this correct? Writing the release as `0.9.0b1` rather than for example `0.9.0-b1`?

Is there a reason for allowing a non-number in the last position (major, minor, patch - note that I'm assuming SemVer here). I've seen something like "MAJOR.MINOR.PATCH-PRERELEASE" where prerelease could be alpha.1 or say beta.1.

Reply:
@consideRatio Prereleases in Python typically do not include dashes. https://www.python.org/dev/peps/pep-0440/